Breakage of x86 (32bit) builds

This commit:
https://github.com/fmtlib/fmt/commit/b98926b73ba35dd750be2a6c1db345e46d775172

dropped the operator overload:
```
  friend constexpr auto operator~(const uint128_fallback& n)
      -> uint128_fallback {
    return {~n.hi_, ~n.lo_};
  }
```

This might have been on purpose, I don't know. But this results in a failure to compile under x86.

Eventual compile error:
```
  /tmp/etcetc/libs/fmt/12.2.0/include/fmt/format.h:3108:14: error: no match for ‘operator~’ (operand type is ‘fmt::v12::detail::uint128’)                          
   3108 |       f.f &= ~(inc - 1);                                                                                                                                                            
        |              ^~~~~~~~~~    
```

Feel free to close if x86 is intentionally not supported anymore.

--- a/include/fmt/format.h
+++ b/include/fmt/format.h
@@ -326,6 +326,9 @@ class uint128 {
       -> uint128 {
     return {lhs.hi_ & rhs.hi_, lhs.lo_ & rhs.lo_};
   }
+  friend constexpr auto operator~(const uint128& n) -> uint128 {
+    return {~n.hi_, ~n.lo_};
+  }
   friend FMT_CONSTEXPR auto operator+(const uint128& lhs, const uint128& rhs)
       -> uint128 {
     auto result = uint128(lhs);

--- a/test/format-test.cc
+++ b/test/format-test.cc
@@ -82,6 +82,11 @@ TEST(uint128_test, minus) {
   EXPECT_EQ(n - 2, 40);
 }
 
+TEST(uint128_test, bitwise_not) {
+  auto n = ~uint128(0x123456789abcdef0, 0x0fedcba987654321);
+  EXPECT_EQ(n, uint128(0xedcba9876543210f, 0xf0123456789abcde));
+}
+
 TEST(uint128_test, plus_assign) {
   auto n = uint128(32);
   n += uint128(10);

